projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -287,7 +287,13 @@ control_service_release:
 
 # TODO: automatically rebuild image on vdk-heartbeat change
 control_service_vdk_heartbeat_release:
-  extends: .images:dind
+  image: docker:19.03.15
```

Comment:
nit: write a comment that are are using older version because ...

Reply:
Done